src/QuadForm/Torsion.jl

```diff
@@ -1695,16 +1701,29 @@ function brown_invariant(T::TorQuadModule)
   return brown
 end
 
+function _as_finite_bilinear_module(T::TorQuadModule)
+  n = modulus_bilinear_form(T)
+  return torsion_quadratic_module(cover(T), relations(T); modulus = n, modulus_qf = n)
+end
+
 @doc raw"""
-    genus(T::TorQuadModule, signature_pair::Tuple{Int, Int}) -> ZZGenus
+    genus(T::TorQuadModule, signature_pair::Tuple{Int, Int};
+                            parity::RationalUnion = modulus_quadratic_form(T))
```

Comment:
The parity must not be a rational number. Hence `RationalUnion` is missleading.

Reply:
Yes, it has to be a rational number. In any case, it should be restricted to 1 or 2 for us, but then if we enter a `Int`, `ZZRingElem` or `QQFieldElem`, then everything fits in `RationalUnion`.